sentry_sdk/utils.py

```diff
@@ -1159,9 +1159,13 @@ def _is_contextvars_broken():
         pass
 
     try:
+        import greenlet  # type: ignore
         from eventlet.patcher import is_monkey_patched  # type: ignore
 
-        if is_monkey_patched("thread"):
+        version_tuple = tuple(
+            [int(part) for part in greenlet.__version__.split(".")[:2]]
+        )
```

Comment:
We should use `parse_version` for parsing the version. This function: https://github.com/getsentry/sentry-python/blob/antonpirker/fix-aws-secrects/sentry_sdk/utils.py#L1514